Bottom: a114bb4aa4a9ac573942168f671e7dc28dade20c
Top:    54ef9ca3b5f1ca0cead35af367732d1081884806
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-14 00:58:46 +0000

Refresh of remove-thindraw_pcb_polygon-fr

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index b09d16b..88b9360 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -2155,6 +2155,7 @@ hid_gtk_init ()
   ghid_hid.draw_arc                 = ghid_draw_arc;
   ghid_hid.draw_rect                = ghid_draw_rect;
   ghid_hid.fill_circle              = ghid_fill_circle;
+  ghid_hid.fill_polygon             = ghid_fill_polygon;
   ghid_hid.fill_rect                = ghid_fill_rect;
 
   ghid_hid.calibrate                = ghid_calibrate;
